src/runner/runESLint.js

```diff
@@ -97,11 +98,51 @@ const runESLint = async ({ testPath, config, extraOptions }) => {
   );
 
   if (report[0]?.errorCount > 0) {
-    return fail({
-      start,
-      end,
-      test: { path: testPath, title: 'ESLint', errorMessage: message },
-    });
+    const startTime = new Date(start).getTime();
+    const endTime = new Date(end).getTime();
+    return {
```

Comment:
having to manually construct the entire result is odd - we should still be able to use `fail` to report many errors, no?

Reply:
Yeah, it's a shame, but [`fail` does not support multiple errors](https://github.com/jest-community/create-jest-runner/blob/main/lib/fail.ts#L46-L48).

[`toTestResults`](https://github.com/jest-community/create-jest-runner/blob/main/lib/toTestResult.ts#L65) actually [does](https://github.com/jest-community/create-jest-runner/blob/main/lib/toTestResult.ts#L50) but `toTestResults` isn't exposed through the main module.

We have a few options:

1. I can put in a PR to `create-jest-runner` to expose `toTestResults` correctly.
2. I can put in a PR to `create-jest-runner` to allow multiple failures in `fail`.
3. We can do an internal require to `create-jest-runner/toTestResults` instead.

My preference would be either to do #3 or to move forward with this PR as is.  I'll follow up by doing #1 AND #2 on `create-jest-runner` and whenever that lands, I'll come back here and put in another PR to bump the version and use the new shininess.

Thoughts?

Reply:
We should do both 1 and 2, yes 👍 I don't think we need 3 - doing it manually like you do here is fine in that case. Perhaps leave a code comment saying so?

Reply:
Added a comment to that effect in the latest commit